pysal/contrib/geotable/utils.py

```diff
@@ -50,13 +49,40 @@ def to_gdf(df, geom_col='geometry', **kw):
     """
     from geopandas import GeoDataFrame
     from shapely.geometry import asShape as sShape
-    df[geom_col] = df[geom_col].apply(sShape)
-    return GeoDataFrame(df, geometry=geom_col, **kw)
+    out = df.copy()
```

Comment:
In order to shield against potential future API changes, should we explicitly make it a deep copy?

`out = df.copy(deep=True)`

`pandas` relevant doc [here](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.copy.html)